label_sync/cluster/label_sync_job.yaml

```diff
@@ -0,0 +1,33 @@
+# Keep in sync with label_sync_cron_job
+# This is a oneoff job used for manual runs.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: label-sync-1008-002
+spec:
+  template:
+    spec:
+      containers:
+        - name: label-sync
+          image: gcr.io/k8s-testimages/label_sync:v20190126-6c4304780
+          args:
+          - --config=/etc/config/labels.yml
```

Comment:
I suggest we change labels.yml  into labels.yaml, since the example is named after .yaml and yaml is used as well. Be consistent in all the context.